vexim/adminfail.php

```diff
@@ -35,6 +35,8 @@
               . '<td align="center">'
               . '<a href="adminfaildelete.php?user_id='
               . $row['user_id']
+              . '&localpart='
```

Comment:
This should really say `&amp;localpart`, although I'm not sure why we need localpart at all, just to delete an alias by its id.